docs/Manuals/Game/shortcut-options.rst

```diff
@@ -0,0 +1,373 @@
+.. SPDX-License-Identifier: GPL-3.0-or-later
+.. SPDX-FileCopyrightText: 2023 James Robertson <jwrober@gmail.com>
+
+.. Custom Interpretive Text Roles for longturn.net/Freeciv21
+.. role:: unit
+.. role:: improvement
+.. role:: wonder
+
+Client Shortcut Options
+***********************
+
+As discussed in the :ref:`Shortcut Options <client-manual-shortcuts>` section of the game manual, there are a
+large collection of keyboard and mouse shortcut options available. This page lists them in detail. Here is a
+sample of the top of the shortcut options dialog.
+
+.. _Shortcut Options Dialog2:
+.. figure:: /_static/images/gui-elements/shortcut-options.png
+  :scale: 65%
+  :align: center
+  :alt: Freeciv21 Shortcut Options dialog
+  :figclass: align-center
+
+  Shortcut Options Dialog
+
+
+To override the default (e.g. change a shortcut setting), click your mouse cursor on the box on the right and
+then type the shortcut you want to set the setting to. For example: You want to change the
+:guilabel:`Scroll Map` option to the :guilabel:`PgUp` key, you would click in the box and then press the
+:guilabel:`PgUp` key on your keyboard. To reset to defaults, you can click on the :guilabel:`Reset` button.
+Click :guilabel:`Save` and then :guilabel:`Close` when finished.
+
+Here are the available shortcuts:
+
+Scroll Map
+  Scroll the map view. You can also use two-finger swipe on a trackpad if you have one (such as on a laptop)
+  in all four directions to scroll the map.
```

Comment:
It is talked about there as well. Since it isn't obvious that it is not just right-click to move around the map I think its worth keeping the duplicate text here.